ilastik/workflows/pixelClassification/pixelClassificationWorkflow.py

```diff
@@ -35,20 +36,28 @@
 from ilastik.applets.featureSelection import FeatureSelectionApplet
 from ilastik.applets.pixelClassification import PixelClassificationApplet, PixelClassificationDataExportApplet
 from ilastik.applets.batchProcessing import BatchProcessingApplet
+from ilastik.utility import SlotNameEnum
 
 from lazyflow.graph import Graph
 from lazyflow.roi import TinyVector, fullSlicing
 
 class PixelClassificationWorkflow(Workflow):
-
     workflowName = "Pixel Classification"
     workflowDescription = "This is obviously self-explanatory."
     defaultAppletIndex = 0 # show DataSelection by default
 
-    DATA_ROLE_RAW = 0
-    DATA_ROLE_PREDICTION_MASK = 1
-    ROLE_NAMES = ['Raw Data', 'Prediction Mask']
-    EXPORT_NAMES = ['Probabilities', 'Simple Segmentation', 'Uncertainty', 'Features', 'Labels']
+    @enum.unique
+    class Roles(SlotNameEnum):
+        RAW_DATA = 0
+        PREDICTION_MASK = enum.auto()
```

Comment:
Good point, I missed the part that it was an implementation detail. Still, I'm returning `0 if count == 0 else last_values[-1] + 1` so that one could use an enum that does not start with `0` and do something like:

```
class MyOutputs(SlotNameEnum):
     OUTPUT1 = 3
     OUTPUT2 = enum.auto()

# this enum continues enumerating from where MyOutputs stopped
class MySpecialOutputs(SlotNameEnum):
     SPECIAL_OUTPUT1 = MyOutputs.getNext()
     SPECIAL_OUTPUT2 = enum.auto()
```